src/integration-test/resources/CasedataIT/__files/test4_PostParkingPermitCaseAndAnge/mappings/casedata_getCase.json

```diff
@@ -0,0 +1,15 @@
+{
+	"request": {
+		"method": "GET",
+		"urlPathPattern": "/case-data/2260/(.+)/errands/24"
+	},
+	"response": {
+		"headers": {
+			"Content-Type": [
+				"application/json"
+			]
+		},
+		"bodyFileName": "test1_PostParkingPermitCase/response/casedata_getCase.json",
```

Comment:
Should this not point to a file in the test04 folder or a "common" folder? Pointing to a file in another tests folder feels wrong.

Reply:
Response should also reflect what was posted post. test1 is missing phase action for example.

Reply:
In test1 phase action is not expected.